src/ISO3166.php

```diff
@@ -137,12 +152,12 @@ public function getIterator()
     private function getBy($key, $value)
     {
         foreach ($this as $country) {
-            if (0 === strcasecmp($value, $country[$key])) {
+            if ($value === $country[$key]) {
                 return $country;
             }
         }
 
-        throw new \OutOfBoundsException('ISO 3166-1 does not contain: ' . $value);
+        throw new \OutOfBoundsException(sprintf('ISO 3166-1 does not contain: %s', $value));
```

Comment:
I would make it more specific though, of you just read that in your logs, you have nothing to go by.